lib/index.js

```diff
@@ -438,11 +404,13 @@ exports.setStatus = brCallbackify(async ({actor, id, status}) => {
  * @param {string} options.id - The ID of the Account that is to be updated.
  * @param {Array} [options.add=[]] - The resourceRoles to add.
  * @param {Array} [options.remove=[]] - The resourceRoles to remove.
- *
+ * @param {number} options.sequence - The sequence number that must match the
+ * current record prior to the patch.
```

Comment:
I guess the lint rules don't complain about lack of indents here?  Should be 2 spaces.

Reply:
yeah, linter doesn't complain. I can fix that though.